src/app.exec.ts

```diff
@@ -38,11 +37,18 @@ export default async function exec(opts: Args) {
     } else if (opts.pkgs.length) {
       await repl(installations, env)
     } else {
-      await help()
-      Deno.exit(1)
+      switch (verbosity) {
+      case Verbosity.quiet: // error message will be eaten
+      case Verbosity.normal:
+        throw new UsageError()
+      default:
+        break // we show version and exit
+      }
```

Comment:
Should have a break on Verbosity.quiet and break on default is unnecessary

Reply:
The reason for this is there is no error for args.length zero (we show usage because typing `tea` by itself should do *something*). In verbose mode though we want to output the “announce” text which is `tea vx.y.z` and the tea binary path.

I did comment that.

So yes, it's right, though needs a comment.

Reply:
> Should have a break on Verbosity.quiet

we want the fall through to throw a usage error.

> break on default is unnecessary

sure but it looks funky.

Reply:
an `if` statement would be less confusing I think. Will change.